libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/cert/impl/JDBCCertRecordStoreConnection.java

```diff
@@ -208,6 +210,7 @@ public boolean updateX509CertRecord(X509CertRecord certRecord) throws ServerReso
             ps.setString(10, certRecord.getProvider());
             ps.setString(11, certRecord.getInstanceId());
             ps.setString(12, certRecord.getService());
+            ps.setString(13, certRecord.getSiaProvider());
```

Comment:
the sia provider is not the last component in the UPDATE command. It should be field 10 and then others are pushed back by 1.

can you also please review the test case for this method since this should have been caught by the unit tests.

Reply:
yeah, good catch.